egs/vctk/TTS/vits/export-onnx.py

```diff
@@ -212,10 +212,15 @@ def export_model_onnx(
     )
 
     meta_data = {
-        "model_type": "VITS",
+        "model_type": "vits",
         "version": "1",
         "model_author": "k2-fsa",
-        "comment": "VITS generator",
+        "comment": "icefall",  # must be icefall for models from icefall
+        "language": "English",
+        "voice": "en-us",  # Choose your language appropriately
+        "has_espeak": 1,
+        "n_speakers": 108,
+        "sample_rate": 22050,  # Must match the real sample rate
```

Comment:
Can we obtain the sample rate from the model?

Is `sample_rate` an attribute of the model?